blocks/article-feed/article-feed.js

```diff
@@ -73,8 +73,10 @@ async function decorateArticleFeed(articleFeedEl, config, offset = 0) {
           ${pictureTag}
         </div>
         <div class="article-card-body">
-        <p class="article-card-category">${category}</p>
-        <h3>${title}</h3>
+          <p class="article-card-category">
+            <a href="${window.location.origin}/blog/categories/${category}">${category}</a>
+          </p>
+          <h3>${title}</h3>
           <p>${description}</p>
         </div>`;
```

Comment:
this repetition makes me think a `buildCard` function that can be shared might be beneficial...